tools/converter/reports_converter.cpp

```diff
@@ -0,0 +1,104 @@
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <vector>
+
+#include "binary_reader/ReadManager.h"
+#include <bbp/sonata/reports.h>
+#include <utils/logger.h>
+
+using namespace bbpReader;
+
+int get_steps_to_write (const char* var_name) {
+    int steps_to_write = 10;
+    char* var_value = getenv(var_name);
+    if (var_value != nullptr && atoi(var_value) > 0) {
```

Comment:
I didn't think it was a parameter that the user cares, is more a developer thing. I saw that in general 10 steps to write is best, but it might depend of the report....

But of course I could add some comments.